scalardl-test/verification-config.toml

```
@@ -24,6 +24,8 @@
   is_verification = true
   num_accounts = 10
   population_concurrency = 32
+  retry_interval_milli_sec = 1000
```

It's not clear where the variables are used since the names are too general.
```suggestion
  checker_retry_interval_millis = 1000
```

---

Sorry, I will update the names to be more specific.
I will update the variable names as specified.